.github/workflows/mmda-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Test with Python ${{ matrix.python-version }}
         run: |
-          pip install -e .[dev,pysbd_predictors,hf_predictors,lp_predictors]
+          pip install -e .[dev,pysbd_predictors,hf_predictors,lp_predictors,grobid_full_text_parser]
```

Comment:
ah nvm i see, we dont actually test running grobid itself here

Reply:
the test mocks hitting grobid by just returning the xml that grobid would return so that it doesn't need grobid at all (I copied the method used for the test for the pre-existing grobid parser)